interface/forms/phq9/common.php

```diff
@@ -0,0 +1,340 @@
+<?php
+
+/**
+ * phq-9 form using forms api     new.php    create a new form
+ *
+ * @package   OpenEMR
+ * @link      http://www.open-emr.org
+ * @author    Ruth Moulton <moulton ruth@muswell.me.uk>
+ * @copyright Copyright (c) 2021 ruth moulton <ruth@muswell.me.uk>
+ *
+ * @license   https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
+ */
+
```

Comment:
this script should be the one to have the globals.php require (to ensure security and ensure it is never called twice in one script run):
```
 require_once("../../globals.php");
 require_once("$srcdir/api.inc");
```